### DISCUSSION TITLE
Error: Valid installation of TensorFlow not found.

### DISCUSSION THREAD

**Discussion by kintarinrl** [teamalgoritma/community#744] [Q&A] [answered]
Error Saat running kode berikut :

![image](https://github.com/teamalgoritma/community/assets/63953994/b569dbf1-4c38-4026-8b71-d72d422d0c50)

**Comment by kintarinrl** [comment]
Sudah uninstall install pillow ada warning seperti berikut
![image](https://github.com/teamalgoritma/community/assets/63953994/1a3f13cd-6008-4fe6-b027-feffe4b76d1e)

**Reply by finesaaa** [reply, marked as answer]
Boleh coba uninstall numpynya : `pip uninstall numpy` dan lalu diinstall numpynya kembali Bu @kintarinrl , rasanya itu karena terdapat cache
